launch_testing/launch_testing/__init__.py

```diff
@@ -35,6 +35,7 @@ def __init__(
         self.name = name
         self.launch_descriptor = launch_descriptor
         self.expected_lines = expected_lines
+        self.expected_output = b'\n'.join(self.expected_lines)
```

Comment:
The functionality that I am trying to achieve with this pull request is supporting a single regex that matches multiple lines.

e.g. Suppose the "pass" condition of my test is see the following lines printed in order

"foo 1
bar 2"

but 1 and 2 can be any number.

Then naturally I would want to match them with a single regex that spans 2 lines, something like "foo \d+\nbar\d+"